gzll/CHANGELOG.rst

```diff
@@ -9,6 +9,17 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Main branch
```

Comment:
Not so sure whether this should be something else, like nRF Connect SDK v2.1.0, for example.

Reply:
Probably not. IMO, it will take a push for v2.2.0.